### PR TITLE
Update systemd service file

### LINKED ISSUE #13: Allow minio to bind to lower ports
By adding this line to the systemd [service] config:
`AmbientCapabilities=CAP_NET_BIND_SERVICE`

You can allow minio to bind to lower ports (e.g. 80 and 443) without running it as root.
Maybe pack it behind a conditional setting.

just fyi.

### PR DESCRIPTION
**Pull request by paulfantom**
- Update service file to newer version from [upstream](https://github.com/minio/minio-service/blob/master/linux-systemd/minio.service)
- Add `ansible_managed` to every template
- Add CAP_NET_BIND_SERVICE to systemd service file when port is lower than 1024 (Resolve #13)

### CODE REVIEW

File: templates/minio.service.j2 [hunk 1]
```diff
@@ -33,5 +39,9 @@ SendSIGKILL=no
 
 SuccessExitStatus=0
 
+{% if minio_server_addr.split(':')[1] < 1024 %}
```
Comment (SuperQ) [inline templates/minio.service.j2]: This will break with v6 :grin:  `[::1]:80`.

Comment (paulfantom) [reply templates/minio.service.j2]: should be fixed now :smile:

**Comment by paulfantom** [comment]
@SuperQ I've added some tests. Unfortunatelly I don't see a way to test ipv6.

**Comment by SuperQ** [comment]
Can you add an assert for the service file `file.contains("AmbientCapabilities=CAP_NET_BIND_SERVICE")`. That way we know the math works.

**Comment by paulfantom** [comment]
Seems redundant, since service won't start without it on port 80, but I'll add it.

**Pull request merged**